database/atlas.go

```diff
@@ -0,0 +1,64 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gobackup/gobackup/helper"
+	"github.com/gobackup/gobackup/logger"
+)
+
+// Atlas database
+//
+// type: atlas
+// uri:
+// exclude_tables:
+// args:
+
+type Atlas struct {
+	Base
+	uri           string
+	excludeTables []string
+	args          string
+}
+
+func (db *Atlas) init() (err error) {
+	viper := db.viper
+
+	db.uri = viper.GetString("uri")
+	db.excludeTables = viper.GetStringSlice("exclude_tables")
+	db.args = viper.GetString("args")
+	return nil
+}
+
+func (db *Atlas) build() string {
+	return mongodumpCli + " " +
```

Comment:
Can we just extend the MongoDB to support Atlas?

Looks like we just need to add `uri` option can support this?

https://github.com/gobackup/gobackup/blob/main/database/mongodb.go#L60

Reply:
Yes, we can of course. I updated code.